src/views/HowToPlay/Layout/StyledInstructionsLayout.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+const StyledInstructionsLayout = styled.div`
+  .container {
+    display: flex;
+    align-items: center;
+    margin: 25px 0 0 3em;
+  }
+
+  h3 {
+    margin: 0;
+    &.highlight-text {
+      margin-left: 0.25em;
+    }
+    justify-content: center;
+    align-items: center;
+  }
+
+  margin-bottom: 3em;
```

Comment:
set a left/right margin here. Leave top and bottom alone, your padding on the KeepReading component takes care of the spacing you need.

```suggestion
  margin-bottom: 0 3em;
```